From a568bbdd10bb5dd543d1177c1ee7f7f9d72bfa9f Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@cloudera.com>
Date: Fri, 23 Oct 2015 15:13:00 -0700
Subject: [PATCH 0940/2848] CDH-33458: HttpClient connection refused in
 TestSecondaryWebUi.testSecondaryWebUiJsp Fixed
 the test case caused by backporting HDFS-3059

Change-Id: Idc6a903b879664677ec11eab92e50ceaae069b60
---
 .../hdfs/server/namenode/TestSecondaryWebUi.java   |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestSecondaryWebUi.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestSecondaryWebUi.java
index db392f2..14c00d7 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestSecondaryWebUi.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestSecondaryWebUi.java
@@ -49,6 +49,7 @@ public static void setUpCluster() throws IOException {
     cluster.waitActive();
     
     snn = new SecondaryNameNode(conf);
+    snn.startInfoServer();
   }
   
   @AfterClass
-- 
1.7.9.5

